fastlane/metadata/de-DE/subtitle.txt

```diff
@@ -1 +1 @@
-Erstelle eine Website oder ein Blog
\ No newline at end of file
+Erstelle deine neue Website
```

Comment:
Before 18.1, I'll submit this and the other subtitle changes for review to GlotPress so I won't have to make them manually anymore every time.